minisom.py

```diff
@@ -71,8 +71,14 @@ def fast_norm(x):
     return sqrt(dot(x, x.T))
 
 
-def asymptotic_decay(learning_rate, t, max_iter):
```

Comment:
Please don't delete this function, users of past versions might still want to use it.